week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,7 +1,60 @@
-function setAlarm() {}
+// Implement the missing functionality according to the task description.
+// Hint: Remember which function you need to call when it's time to alarm.
+  function setAlarm() {
+  let seconds = document.querySelector("#alarmSet").value;
+  document.getElementById("set").addEventListener("click", setup);
```

Comment:
```suggestion
  document.getElementById("set").addEventListener("click", setup);
```
This code here means you have to click on "Set alarm" twice in order to start the count down

Reply:
ohh what a silly mistake! Thank you mitch.